### ISSUE TITLE
Reading: Leveraging Contextual Embeddings for Detecting Diachronic Semantic Shift

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper#69]
## 0. Paper
Now this paper is available in ArXiv. [[link](https://arxiv.org/abs/1912.01072)]

## 1. What is it?
They adopted BERT for the task of diachronic semantic shift.

## 2. What is amazing compared to previous works?
They achieved comparable results to current state-of-the-art works.
* without domain adaptation, (use only its training data, not using external datasets)
* multilingual (immigration news, English-Slovenian)

## 3. Where is the key to technologies and techniques?
Adopt pretrained BERT model (English-BERT-base)
* 12 attention layer
* hidden layer size: 768
* BPE: max-length 256

## 4. How did evaluate it?
LiverpoolFC corpus, Brexit news and Immigration news(multilingual).
<img width="395" alt="スクリーンショット 2020-04-17 18 11 47" src="https://user-images.githubusercontent.com/45454055/79553078-e9ace400-80d6-11ea-9b02-bb7b76a663e2.png">

### LiverpoolFC
* split two corpora (2011-2013, -2017)
* annotated 97 words changed its meaning or not
* there are three types of semantic shifts (metonymic shifts, metaphorical shifts, and meme)
* evaluated by correlation with gold set. (**Quantitative evaluation**)

### Brexit news
* The yearly splits are made on June 24 each year.
* **No evaluation sets**, measuring how does a specific word of interest semantically correlate to other seed words. (**Qualitative evaluation**)
* filtering according to the **normalized Levenshtein distance**. (bigger than 0.5 are discarded)
<img width="458" alt="スクリーンショット 2020-04-17 19 17 43" src="https://user-images.githubusercontent.com/45454055/79559094-1e716900-80e0-11ea-8fdf-d7bf0d41192d.png">

* after that, 50 words are captured by cosine similarity.
* finally, most related ten words are obtained by:
<img width="531" alt="スクリーンショット 2020-04-17 19 22 54" src="https://user-images.githubusercontent.com/45454055/79559512-d737a800-80e0-11ea-947a-69ea74ab58b4.png">

### Immigration news
* shuffled two languages, and devided into five yearly periods (December 31).
* **No evaluation sets**, measuring how does a specific word of interest semantically correlate to other seed words. (**Qualitative evaluation**)

> The hypothesis that justifies this type of evaluation comes from structural linguistics and states that word meaning is a relational concept and that words obtain meaning only in relation to their neighbours (Matthews and Matthews, 2001)

**Comment by a1da4** [comment]
#71 short-term meaning shift
create datasets for this paper
